tools/inference.py

```diff
@@ -42,6 +38,7 @@ def get_args() -> Namespace:
     parser.add_argument("--weight_path", type=str, required=True, help="Path to a model weights")
     parser.add_argument("--image_path", type=str, required=True, help="Path to an image to infer.")
     parser.add_argument("--save_path", type=str, required=False, help="Path to save the output image.")
+    parser.add_argument("--meta_data", type=str, required=False, help="Path to JSON file containing the metadata.")
```

Comment:
should we use type `Path` here to automatically convert everything to pathlib.Path object to make it easier to work with the path?